test/README.md

```diff
@@ -0,0 +1,225 @@
+
+# Resonate API Testing
+
+These are tests for the WIP Resonate API (v4) migration. This iteration of the tests development simply migrates tests as-built to a Mocha environment and begins development of new tests of the WIP api specification to-be. This stage of the testing effort deliberately keeps things simple. Later iterations will focus on more fancy stuff.
+
+Some clarification is needed regarding the test target. There are two possible targets:
+* http://localhost:4000/api/v3 (this repo: api(v4))
+* https://stream.resonate.coop/api/v3/ (beam/api)
+
+The purpose of these tests is to verify / validate the endpoints of this API, and to begin development of a more robust API test suite.
+
+For the sake of total clarity, these tests live in the `/api/test` directory. The previous tests live in `/api/test/as-built` directory. The new tests live in `api/test/to-be` directory. In the `api/test/to-be` directory there are three subfolders (Admin.ts, Api.ts, and User.ts), each of which corresponds to a file in the `beam/src/services` folder.
+
+These tests expect a running instance of the Docker container that you get when you `docker compose up` at the `api` repo root. Make sure first that this container instance is running / has finished loading / etc. before running the tests. 
+
+You also need to seed the database.
+
+The `Getting started with local dev` section of the main repo [README.md](../README.md) file explains what to do.
+
+## Setup and running
+There is a file name `Template.test.js` in the `test` folder. You can copy this file to start writing new tests more quicly.
+
+There is a file named `testConfig.js` in the `test` folder. It has vars and modules that are used in test files. It's basically used as an include header at the head of each test file. If you create new tests, this file will help you go faster.
+
+In `testConfig.js` there is a const `baseURL`. This is set to `http://localhost:4000/api/v3` so that you can run the tests against a local Dockerized resonate api (v4) container instance. If you need to run the tests against a different instance of the API, you can comment this const out and replace it with a url of your choosing, ie `const baseURL = 'http://awesome.sauce.com/api/v2'` or similar.
+
+`https://stream.resonate.coop/api/v3/` is the 'beam/api' instance. This is another possible test target. Further clarification of the correct test target url is needed. You can change the baseURL const in `testConfig.js` to this URL if that is appropriate.
+
+Run the old / as-built tests
+```sh
+npm run test:as-built
```

Comment:
Since we use yarn as our package manager, we should be consistent. 

```
yarn test:as-built
```
Same applies to all the other proposed commands below.

Reply:
ah, right. My bad.  Just habit; thanks for catching that!